tests/preconversion_assessment/test_output_schema.py

```diff
@@ -0,0 +1,14 @@
+import jsonschema
+import json
+from scripts.preconversion_assessment_script import OutputCollector
+
+
+def test_output_schema():
+    """Test that pre-conversio nreport is wrapped in expected json"""
+    output_collector = OutputCollector()
+    empty_output = output_collector.to_dict()
+
+    with open("schemas/preconversion_assessment_schema_1.0.json", "r") as schema:
+        schema_json = json.load(schema)
+    # If some difference between generated json and its schema invoke exception
+    jsonschema.validate(instance=empty_output, schema=schema_json)
```

Comment:
This currently fails, there is question if it's indeed wrong or if we are missing some information

I assumed this can be used as example of expected input https://gist.github.com/pr-watson/59b2fefdef610fa1ceaa1fb5de1d5112  but current script doesn't produce `alert` property but has `status` property.

Expected:
```
{
      alert: true,
      report: '',
      message: 'Your system has 2 inhibitors out of 3 potential problems.',
      report_json: {
        entries: [
          ...
        ],
      }
}
```

Actual:
```
{
      status: "some string",
      report: '',
      message: 'Your system has 2 inhibitors out of 3 potential problems.',
      report_json: {
        entries: [
          ...
        ],
      }
}
```
Note that the status the script produces seems to be one level above in the given gist

Reply:
@pr-watson @r0x0d What is the correct output? am I missing something? Is the gist right? How should I generate the alert - what would be the rule for the True or False value? If the script is right, where does the alert comes from in the gist?

Reply:
@andywaltlova the alert field was used in previous leapp tasks but after discussions with Dan, Michael and some convert2RHEL members we decided to not use the alert field anymore

Reply:
Alright, talked with @pr-watson , script correct - gist old (leapp format)